cmd/gogio/androidbuild.go

```diff
@@ -153,10 +154,15 @@ func buildAndroid(tmpDir string, bi *buildInfo) error {
 	case "archive":
 		return archiveAndroid(tmpDir, bi, perms)
 	case "exe":
-		if err := exeAndroid(tmpDir, tools, bi, extraJars, perms); err != nil {
+		if err := exeAndroid(tmpDir, tools, bi, extraJars, perms, false); err != nil {
 			return err
 		}
 		return signAPK(tmpDir, tools, bi)
+	case "bundle":
```

Comment:
Can we avoid another buildmode for what is arguably a variation of "exe"? How about choosing aab mode according to the output filename, so that `gogio -o blah.aab` will output in AAB format? There's precedent in that `-target ios` selects ".ipa" vs ".app" format according to output filename.